runtime/jvmti/jvmtiStartup.c

```diff
@@ -446,45 +447,50 @@ issueAgentOnLoadAttach(J9JavaVM * vm, J9JVMTIAgentLibrary * agentLibrary, const
 }
 
 /**
- * Load a JVMTI agent and call the agent's initialization function
- * @param vm Java VM
- * @param agentLibrary environment for the agent
- * @param name of the initialization function
- * @return JNI_ERR, JNI_OK
+ * Load a JVMTI agent and call the agent's initialization function.
+ *
+ * @param[in] vm Java VM
+ * @param[in] agentLibrary environment for the agent
+ * @param[in] loadFunctionName name of the initialization function
+ * @param[in] loadStatically a boolean indicating if the library is to be loaded statically
+ * @param[in/out] found a pointer to a boolean indicating whether the load function was found
+ * @param[in/out] errorMessage the error message when opening the shared library
```

Comment:
nit: the description for errorMessage is missing "a pointer to" ...

Reply:
Added